src/components/nav.js

```diff
@@ -1,22 +1,29 @@
 import PropTypes from "prop-types"
-import React from "react"
+import React, { useState, useEffect } from "react"
 import Logo from "./logo"
-import styled from "styled-components"
+import { SectionContent } from "../styles/Section"
+import styled, { css } from "styled-components"
 
-const Container = styled.div`
+const Container = styled.nav`
   position: fixed;
-  height: 48px;
   width: 100%;
-  display: flex;
-  justify-content: space-between;
   z-index: 100;
   background: rgba(255, 255, 255, 0.9);
```

Comment:
I think you could also put the interpolation in the opacity field of the background. Have it return 0 if props.hideBackground, and 0.9 otherwise.

Reply:
I'm not sure which way is better practice

Reply:
I'm gonna go with using an `opacity` interpolation is better practice, because we aren't changing the background color at all, which is what this currently suggests.

Reply:
Turns out `opacity` will remove the entire nav, which we don't want. I guess it is actually the background that is changing, which is what this suggests (forgot it wasn't the `background-color` property, which would be different).